code/modules/admin/admin_verbs.dm

```diff
@@ -191,7 +191,8 @@ GLOBAL_PROTECT(admin_verbs_debug)
 	/client/proc/check_timer_sources,
 	/client/proc/toggle_cdn,
 	/client/proc/cmd_give_sdql_spell,
-	/client/proc/adventure_manager
+	/client/proc/adventure_manager,
+	/client/proc/hard_deletion_toggle
```

Comment:
```suggestion
	/client/proc/hard_deletion_toggle,
```
Trailing comma good

Reply:
doesn't that compile error on byond?

Reply:
nah

Reply:
It adds a null to the list though IIRC, right?

Reply:
Seemingly not if used at the very end of the list 
![image](https://user-images.githubusercontent.com/58055496/123375635-f714f800-d53d-11eb-815b-c5f0f46a8c3c.png)

Reply:
In fact the rule appears to be ignore trailing commas

Reply:
It does not add a null entry.